prac_05/emails.py

```diff
@@ -0,0 +1,147 @@
+"""
+CP1404/CP5632 Practical 05
+Email Register
+"""
+
+
+def main():
+    """This program allows you to store multiple emails, predict and confirm names upon entry."""
+    email_register = {}
+    email = " "
+    while email != "":
+        try:
+            email = input("Email: ")
+            if email != "":
+                validate_email(email)
+                predicted_name = extract_full_name(email)
+                accept_predicted = input("Is your name {}? (Y/n) ".format(predicted_name)).upper()
+                name = validate_name(predicted_name, accept_predicted)
+                # print("Stored name:", name)
+                store_in_email_register(email_register, name, email)
```

Comment:
In the current code, does this need to be a specific function?

Reply:
I can see 4 functions from your selected lined 11-20 I don't know which on you mean.